Voronoi_diagram_2/include/CGAL/Delaunay_triangulation_on_sphere_adaptation_policies_2.h

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2021 GeometryFactory
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+//
+// Author(s)     : Menelaos Karavelas <mkaravel@iacm.forth.gr>
+//                 Mael Rouxel-Labbé
+
+#ifndef CGAL_DELAUNAY_TRIANGULATION_ON_SPHERE_ADAPTATION_POLICIES_2_H
+#define CGAL_DELAUNAY_TRIANGULATION_ON_SPHERE_ADAPTATION_POLICIES_2_H 1
```

Comment:
do we have any other file with the `1`?

Reply:
Quite a lot:
```shellsession
% ack '# *define.*_H *1' */include -l | wc -l      
227
```

Reply:
less than 7% of the include files. I do not really care, it just caught my eye and I directly thought of a special case when writing scripts :)